src/world.rs

```diff
@@ -225,19 +265,30 @@ impl World {
     }
 
     /// Adds a cube and then recomputes the visibility of the affected cubes (neighbors)
-    fn add_cube(&mut self, at: Vector3, block: Block) {
+    /// Return the cube that need to be rendered 
+    /// and the position of the rendered cube that need to be destroyed
+    fn add_cube(&mut self, at: Vector3, block: Block) -> (Vec<Vector3>,Cube) {
         // For all the neighbors positions, increase their internal counter
         let mut count = 0;
+        let mut to_destroy = Vec::new();
```

Comment:
It's not `to_destroy`, but `to_hide`

Reply:
I'm sorry about all these comments about naming, but it is really important when working in a team.